plugins/content_distribution/admin/forms/ConfigurableProfileConfiguration.php

```diff
@@ -5,6 +5,23 @@
  */
 abstract class Form_ConfigurableProfileConfiguration extends Form_ProviderProfileConfiguration
 {
+	public function init ()
+	{
+		parent::init();
+		
+		$this->addElement('checkbox', 'exclude_legacy_categories', array(
+			'label' => 'Exclude legacy categories',
```

Comment:
It's not really that either, sine entries can be associated with categories without PC via the categoryEntry objects as well.